### DISCUSSION TITLE
When skins are used, the color of the text underlay does not match the overall color of the window

### DISCUSSION THREAD

**Discussion by churchill123** [digao-dalpiaz/Dam#35] [Ideas]
Also, the appearance of the buttons does not match the appearance of the buttons from the skin

![Безымянный1](https://github.com/digao-dalpiaz/Dam/assets/26910229/201110c5-424b-49f7-94d5-c82bd930d713)
![Безымянный2](https://github.com/digao-dalpiaz/Dam/assets/26910229/d2ea94f3-fa92-45f3-927a-f57bdf0be2e4)

**Comment by churchill123** [comment]
I propose the following translation of Russian-language strings in file **DamLang.ini**

[Russian]
OK=Ок
Yes=Да
No=Нет
Info=Информация
Quest=Вопрос
Warn=Предупреждение
Error=Ошибка
Msg=Сообщение

**Reply by digao-dalpiaz** [reply]
Done!
